de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/HTMLWhiteboardView.java

```diff
@@ -37,5 +45,47 @@ public void createPartControl(Composite parent) {
     browser = new Browser(parent, SWT.NONE);
     browser.setJavascriptEnabled(true);
     LOG.debug("HTML Whiteboard Browser Started");
+
+    // open the whiteboard document in browser
```

Comment:
This is done in line 75.

The code below the comment tries either to find the file or to extract it from the jar file (line 56)